moment-range/index.d.ts

```diff
@@ -1,46 +1,57 @@
-// Type definitions for Moment.js 2.0.4
+// Type definitions for Moment-range.js 3.0.3
 // Project: https://github.com/gf3/moment-range
-// Definitions by: Bart van den Burg <https://github.com/Burgov>, Wilgert Velinga <https://github.com/wilgert>
+// Definitions by: Bart van den Burg <https://github.com/Burgov>, Wilgert Velinga <https://github.com/wilgert>, Juan Francisco Adame <https://github.com/franjuan>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import * as moment from 'moment';
 
 export = moment;
 
+type interval = 'years' | 'quarters' | 'months' | 'weeks' | 'days' | 'hours' | 'minutes' | 'seconds' | 'milliseconds';
+
 declare module 'moment' {
      interface Range {
         start: Moment;
         end: Moment;
 
-        contains (other: Date, exclusive?: boolean): boolean;
-        contains (other: Moment, exclusive?: boolean): boolean;
+        contains (other: Range, options?: {exclusive?: boolean}): boolean;
 
-        overlaps (range: Range): boolean;
+        overlaps (range: Range, options?: {adjacent?: boolean}): boolean;
 
-        intersect (other: Range): Range;
+        intersect (other: Range, options?: {adjacent?: boolean}): Range;
```

Comment:
does not look like `intersect` takes a second argument, see https://github.com/gf3/moment-range/blob/476d93cd1218d79f4d774ec6296aed102fb54a3d/lib/moment-range.js#L154